Doxyfile

```diff
@@ -104,7 +104,8 @@ EXCLUDE_PATTERNS       = */build/* \
                          *_impl.hpp \
                          *.cpp \
                          *.cc \
-                         *_test.cc
+                         *_test.cc \
```

Comment:
Shouldn't this be `_test.cpp`?  I know you didn't write that code originally but maybe it doesn't hurt to fix. :)